ssas/groups.go

```diff
@@ -199,7 +200,7 @@ func cascadeDeleteGroup(group Group) error {
 	)
 	defer Close(db)
 
-	err := db.Table("systems").Where("group_id = ?", group.GroupID).Pluck("ID", &systemIds).Error
+	err := db.Table("systems").Where("group_id = ?", group.GroupID).Pluck("id", &systemIds).Error
```

Comment:
v2 is now case sensitive?

Reply:
Ah, it's not. I think this change was part of me troubleshooting some other errors. I'l leave it as is though if that's OK with you